### PR TITLE
Create thumbnail image for only a single ingtegration

### LINKED ISSUE #728: generate_preview_images needs not to make so many thumbnails
Currently the `generate_preview_images.py` module, which creates preview images and thumbnails for all datasets in the filesystem, creates thumbnails for all integrations of an observation.  In some observations, this number can be in the thousands, resulting in thousands of thumbnails (e.g. `jw00601/jw00601012001_gs-fg_2018221005716_uncal_integ1000`).  Since the JWQL app only uses thumbnails to serve as buttons to the `view_image` webpages, having thumbnails for more than one integration is unnecessary.

This module should be modified to only create thumbnails for one integration, when applicable.

### PR DESCRIPTION
**Pull request by bhilbert4**
Resolves #728 

This PR makes a small change to `preview_images.py` such that for exposures containing more than one integration, it creates a thumbnail image for only a single integration.

### CODE REVIEW

**Comment by bhilbert4** [comment]
@bourque I think this is ready to go. `generate_preview_images.py` will skip over any existing preview images, so in order to test this, we'd have to delete some images, or perhaps wait until #726 is merged, at which point there will be a whole pile of new data that needs preview images created?

**Comment by bourque** [comment]
Thanks @bhilbert4!  Yes, let's table the testing this until after #726.

**Review by bourque — APPROVED** [review]

**Comment by bourque** [comment]
Thank you @bhilbert4!  On second thought, I want to bring these changes into my branch for #726 so that I can test them there, so I will merge this in now.  If for some reason there are errors, we can deal with them in #726.

**Pull request merged**
